scripts/globals/spells/trust/uka_totlihn.lua

```diff
@@ -5,34 +5,77 @@ require("scripts/globals/trust")
 -----------------------------------
 local spellObject = {}
 
+local jobs1 =
+{
+    xi.job.WHM,
+    xi.job.RDM,
+    xi.job.SCH,
+    xi.job.PLD,
+}
+
 spellObject.onMagicCastingCheck = function(caster, target, spell)
     return xi.trust.canCast(caster, spell)
 end
 
 spellObject.onSpellCast = function(caster, target, spell)
+    -- Add synergy variable to the master
+    local synergy = caster:getLocalVar("mumorSynergy")
+
+    caster:setLocalVar("mumorSynergy", synergy + 1)
+
     return xi.trust.spawn(caster, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.teamworkMessage(mob, {
-        [xi.magic.spell.ULLEGORE] = xi.trust.message_offset.TEAMWORK_1,
-        [xi.magic.spell.MUMOR] = xi.trust.message_offset.TEAMWORK_2,
+        [xi.magic.spell.MUMOR   ] = xi.trust.message_offset.TEAMWORK_1,
+        [xi.magic.spell.ULLEGORE] = xi.trust.message_offset.TEAMWORK_2,
     })
 
-    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 50, ai.r.JA, ai.s.HIGHEST_WALTZ, xi.ja.CURING_WALTZ)
+    -- Locals for synergy
+    local master  = mob:getMaster()
+    local synergy = master:getLocalVar("mumorSynergy")
 
-    mob:addSimpleGambit(ai.t.TARGET, ai.c.NOT_STATUS, xi.effect.FINISHING_MOVE_5, ai.r.JA, ai.s.SPECIFIC, xi.ja.QUICKSTEP, 20)
+    -- Dynamic modifier that checks variable on tick to apply
+    mob:addListener("COMBAT_TICK", "UKA_TOTLIHN_CTICK", function(mobArg)
+        if synergy >= 2 then
```

Comment:
I don't think this synergy variable is guaranteed to be captured for execution in this listener. You'll need to look it up from scratch entirely within the function

Reply:
I will revise this.